stream-chat-android-offline/src/test/java/io/getstream/chat/android/livedata/repository/MessageRepositoryTest.kt

```diff
@@ -14,7 +14,7 @@ import org.junit.runner.RunWith
 
 @RunWith(AndroidJUnit4::class)
 internal class MessageRepositoryTest : BaseDomainTest() {
-    val repo by lazy { chatDomainImpl.repos.messages }
+    val repo by lazy { chatDomainImpl.repos }
```

Comment:
Let's add a task for reviewing the test structure when we're done with all the production code refactoring - for example, this `MessageRepositoryTest` class won't make much sense at that point.